pkg/controllers/management/kontainerdrivermetadata/metadata_handler.go

```diff
@@ -107,9 +101,12 @@ func (m *MetadataController) sync(key string, setting *v3.Setting) (runtime.Obje
 		return nil, fmt.Errorf("invalid number %v", interval)
 	}
 
-	if interval > 0 {
-		logrus.Infof("Refreshing driverMetadata in %v minutes", interval)
-		m.Settings.Controller().EnqueueAfter(setting.Namespace, setting.Name, time.Minute*time.Duration(interval))
+	// Enqueue refresh only on the Leader pod
```

Comment:
Why do we only want to enqueue refresh on the leader pod? I thought the issue is that all pods need to refresh.

Reply:
my understanding was this Enqueue was scheduled as a cluster wide object and would trigger sync across all handlers

Reply:
Looping in @MbolotSuse since I'm not positive but from my understanding enqueueing an object only happens for whatever process the controllers are currently running in. Also curious about whether or not there is a better place to put this controller now that it is running across all replicas.

Reply:
Removed the `onLeader` part so this now runs on all pods